tests/test_serialization.py

```diff
@@ -72,7 +72,7 @@ class ResponseData(Injectable):
 
     @attrs.define
     class MyWebPage(WebPage):
-        url: ResponseUrl
+        url_: ResponseUrl
```

Comment:
It turned out `WebPage.url` already exists, as a shortcut property.

Reply:
Would it make sense to use ItemPage instead? It looks a bit strange to define 2 url attributes, with similar names.